### PR TITLE
[CDAP-14085] Center search box in metadata home page

### PR DESCRIPTION
**Pull request by elfenheart**
JIRA: https://issues.cask.co/browse/CDAP-14085

### CODE REVIEW

File: cdap-ui/app/tracker/templates/main.html [hunk 1]
```diff
@@ -15,17 +15,12 @@
...
-        <span class="fa fa-search form-control-feedback"></span>
-      </div>
-    </div>
-    <cask-resource-center-button></cask-resource-center-button>
+  <div class="form-group has-feedback tracker-search" ng-keypress="MainController.search($event)">
+    <input type="text"
+           class="form-control"
+           placeholder="Search datasets"
+           ng-model="MainController.searchQuery"
+           pattern="^(?![*]).*$">
```
Comment (tonybach) [inline cdap-ui/app/tracker/templates/main.html:23]: I know this forbids the user from searching for `*`, but what's the reason behind that again?

Comment (elfenheart) [reply cdap-ui/app/tracker/templates/main.html:23]: expensive scan

**Review by tonybach — APPROVED** [review]
LGTM 👍

**Comment by elfenheart** [comment]
Build: https://builds.cask.co/browse/CDAP-UDUT37

**Comment by bdmogal** [comment]
LGTM, with the understanding that we may have to do some work on this, because it looks super empty today. @leacd FYI.

**Pull request merged**